packages/backend/src/forums/dto/create-forumcomment.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsDefined, IsMongoId, IsNotEmpty } from 'class-validator';
+
+export class CreateForumCommentDto {
+  @IsNotEmpty({
+    message: 'Text should not be empty.',
+  })
+  @IsDefined({
+    message: 'Text is missing.',
+  })
+  text!: string;
+
+  @IsMongoId({
+    message: 'ParentId should be a valid mongo Id.',
+  })
+  parentId!: string;
```

Comment:
Maybe call this forumId?

Reply:
or postId

Reply:
Actually if comment can be nested under anything, parentId might be the best name here...
If that's the case, we should also probably save in the model what type the parent is?